### PR TITLE
Feature/lga 2411 format a11y json

### PR DESCRIPTION
**Pull request by psweeting1**
## What does this pull request do?

Formatting axe.result dictionary file to only report points of interest in the a11y.json file. Report now only includes violations found in accessbility testing. This has been changed into a json format as well which has better readability.

After all of the tests have been run, the filter_accessibility_report() function creates a second  a11y_filtered.json which filters out and duplicated accessibility violations the report has found. 

## Any other changes that would benefit highlighting?

- Updated README.md file to include how to use accessibility tests in cla_end_to_end
- Updated Git Ignore to stop trying to include downloaded CSV files from tests into the repo
- Removed redundant class variable `accessibility_check = False`  

## Checklist

- [LGA-2411](https://dsdmoj.atlassian.net/browse/LGA-2411)

[LGA-2411]: https://dsdmoj.atlassian.net/browse/LGA-2411?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Pheonnexx — DISMISSED** [review]
LGTM

File: behave/features/steps/common_steps.py [hunk 1]
```diff
@@ -294,13 +295,46 @@ def make_dir(dir):
...
+            axe.write_results(result_format, f"{context.a11y_reports_dir}/a11y.json")
+
     return len(results["violations"]) == 0
+
+
+def filter_accessibility_report(context):
+    with open(f"{context.a11y_reports_dir}/a11y.json", "r") as f:
+        data = json.load(f)
+
+    results_to_copy = []
```
Comment (sarahyjja) [inline behave/features/steps/common_steps.py:329]: Could we rename `results_to_copy` with something more descriptive? _E.g.: `filtered_issues` (or else)_

File: behave/features/steps/common_steps.py [hunk 2]
```diff
@@ -294,13 +295,46 @@ def make_dir(dir):
...
+    for error in data:
```
Comment (sarahyjja) [inline behave/features/steps/common_steps.py:330]: Could we rename `error` with something more descriptive? _E.g.: `step_error` (or else)_

**Review by sarahyjja — CHANGES_REQUESTED** [review]

**Review by sarahyjja — APPROVED** [review]
LGTM

**Review by sarahyjja — APPROVED** [review]

**Review by Pheonnexx — APPROVED** [review]
LGTM 🚀

**Pull request merged**